main.cpp

```diff
@@ -0,0 +1,147 @@
+#include <iostream>
+#include <string>
+#include <stack>
+#include <vector>
+#include <map>
+
+/*
+ ________  _________  ___  _____  _____   _____
+|  ___|  \/  || ___ \/ _ \/  __ \|  ___| /  __ \ _     _
+| |__ | .  . || |_/ / /_\ \ /  \/| |__   | /  \/| |_ _| |_
+|  __|| |\/| ||    /|  _  | |    |  __|  | |  |_   _|_   _|
+| |___| |  | || |\ \| | | | \__/\| |___  | \__/\|_|   |_|
+\____/\_|  |_/\_| \_\_| |_/\____/\____/   \____/
+
+*/
+
+/**
+ * @brief Get and pop last two items on the stack
+ *
+ * @param stack
+ * @param a
+ * @param b
+ */
+void getTwoTopItems(std::stack<int> &stack, int &a, int &b)
+{
+    a = stack.top();
+    stack.pop();
+    b = stack.top();
+    stack.pop();
+}
+
+void loop(std::stack<int> &stack, std::map<std::string, void (*)(std::stack<int> &stack)> &options, std::string &input)
```

Comment:
void pointers, please clean up

Reply:
That's not a void pointer that's a function pointer
And specifically pointer to a function that returns `void`